### PR TITLE
Ready to review

### PR DESCRIPTION
**Pull request by danmontielh**

### CODE REVIEW

**Review by inhaq — CHANGES_REQUESTED** [review]
### Good work on your first project 🏆
I must say you're doing amazing work individually. Here are a few things to fix and improve:

 - Fix your errors by validating it [here](https://validator.w3.org)
 - Improve navbar design by aligning icons properly

Once fixed, kindly resubmit and add me(Inzi) as project reviewer. Happy Coding 😊

**Comment by danmontielh** [comment]
i have improved mi navbar and i don´t have any error in  Markup Validation Service

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,237 @@
+.container {
+    display: grid;
+    grid-gap: 30px;
+    grid-template-columns: 60% 40%;
+    grid-template-rows: 30px auto 200px;
+    padding: 30px;
+}
+
+
+div.header .wrapper {
```
Comment (inhaq) [inline css/style.css]: `div.header` should be `.header` only

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    display: grid;
+    grid-template-columns: repeat(3, auto);
+}
+
+div.header .wrapper .logo i {
```
Comment (inhaq) [inline css/style.css]: again here, don't need to use `div` with class names

**Review by inhaq — APPROVED** [review]
### Great work on your first project 🏆
The design looks good and the code is clean as well. One thing I would like to mention here for your next projects, you don't need to use `div` with CSS selectors. I'll recommend to fix it for this project as well and then submit project completion form. No need to resubmit for review.
###  It's a 👍🏻for this project 🎉

**Pull request merged**